lib/util.js

```diff
@@ -1234,7 +1234,7 @@ function isPrimitive(arg) {
 }
 
 function pad(n) {
-  return n < 10 ? `0${n.toString(10)}` : n.toString(10);
+  return n < 10 ? `0${n}` : n.toString();
```

Comment:
At this point, we can likely just use `padString()` ... e.g.

```js
function pad(n) {
  return String(n).padStart(2, 0);
}
```

Reply:
Fixed it, sorry to forget that!
Many thanks!